### PR TITLE
fix: zoom to layer functionality

### PR DESCRIPTION
**Pull request by nina992**
# Overview
When pressing zoom to layer and then scrolling out or moving the camera out of the layer , then trying to zoom again , the zoom button doesn't work.
## What I've done
adding state to fix that

### CODE REVIEW

File: src/components/molecules/Visualizer/hooks.ts [hunk 1]
```diff
@@ -75,6 +77,7 @@ export default ({
 }) => {
   const engineRef = useRef<EngineRef>(null);
   const [overriddenSceneProperty, overrideSceneProperty] = useOverriddenProperty(sceneProperty);
+  const [, zoomToLayer] = useZoomedLayerId();
```
Comment (KaWaite) [inline src/components/molecules/Visualizer/hooks.ts:80]: So somewhere else we are doing `const [zoomedLayerId] = useZoomedLayerId();`, and then here you are doing `const [, zoomToLayer] = useZoomedLayerId();`
I feel like it makes more sense to declare them in the same place (not sure here or where `zoomedLayerId` is currently declared is better. Please look at the code and decide which is better)

File: src/components/molecules/Visualizer/hooks.ts [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { useRef, useEffect, useMemo, useState, useCallback, RefObject, useReduce
 import { initialize, pageview } from "react-ga";
 import { useSet } from "react-use";
 
+// eslint-disable-next-line no-restricted-imports
```
Comment (KaWaite) [inline src/components/molecules/Visualizer/hooks.ts:6]: Our import restrictions are pretty specific and deliberate. Please don't ignore them.

File: src/components/molecules/Visualizer/hooks.ts (hunk 1 above)
Comment (KaWaite) [reply src/components/molecules/Visualizer/hooks.ts:80]: Please see above comment about import restrictions.

File: src/components/molecules/Visualizer/hooks.ts (hunk 2 above)
Comment (nina992) [reply src/components/molecules/Visualizer/hooks.ts:6]: I noticed that they ignord ather place ,so I continued with that

**Review by KaWaite — APPROVED** [review]

**Pull request merged**